client/Queries/Config/index.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import gql from 'graphql-tag';
+
+export const GET_CONFIG = gql`
+  query {
+    client {
+      about {
+        version
+      }
+    }
+    featureFlags {
+      client {
+        Home {
```

Comment:
Should this be a result of introspection? Or alternatively, in gql something like

`client: [featureFlag]`
`type featureFlag = {name:value}`

Perhaps with nesting (so we can define a shape)?

Reply:
Agreed, feature flags are orthogonal to introspection.